### ISSUE TITLE
Meetup rebranding

### ISSUE THREAD

**Issue by nizaroni** [SunshineJS/undefined#7]
After speaking to Stewart, I'm not 100% sure if we should go with `MiamiJS` or a more South Florida style meetup name. We all agree that we want to be inclusive of all things JavaScript (including browser JavaScript). Most of us also agree that `NodeJS Enthusiasts SE Florida` is a but too wordy. Ideally our brand would imply the general South Florida region without necessarily limiting it to a single city or county (although Miami might be iconic enough to overcome that). That said, we also want an attractive, concise name that's fun to say. Here are the ideas we've thrown around so far:
- `MiamiJS` (iconic, but ultimately only a single city)
- `FortJS` (sounds cool, but again only a single city)
- `SoFloJS`
- `GatorJS`
- `SunshineJS` (I think this is missing the `South` part)
- `GladesJS`

Any input is welcome!

**Comment by nizaroni** [comment]
In the meantime we could use a transitional name. Here are some ideas using the `undefined` concept (we would still transfer to the MiamiJS organization):
- `UndeFLinedJS`
- `undefined`

**Comment by CristinaSolana** [comment]
Personally, don't see a point in going from "NodeJS Enthusiasts SE Florida" to <insert name here> and then again <insert name here>. Once a name is decided on, change it.

**Comment by nizaroni** [comment]
@CristinaSolana Yeah, it might be too much change too fast. Best to make a single a change.

`SunshineJS` has been growing on me lately.

**Comment by luccastera** [comment]
I like SunshineJS.

**Comment by CristinaSolana** [comment]
If SunshineJS turns out to be the one, lmk and I will update the logo to replace Miami with Sunshine. I still like MiamiJS personally, but SunshineJS would be my second pick.

**Comment by young-steveo** [comment]
From the outside looking in:

`undefined` is actually a pretty cool name for a javascript meetup.
1. The `undefined` value is notoriously associated with javascript development
2. It implies that the meetup is not boxed into a specific topic, but it still applies to JS
3.  It sounds cool.  "`undefined`: the south florida javascript meetup"  I could see that on a tee shirt with a slick logo.

**Comment by onel0p3z** [comment]
@young-steveo +1 :+1: thank you for the feedback :D

**Comment by nizaroni** [comment]
Hey guys!

I think we've let this simmer enough. From my perspective, if we are trying to build a family of meetups in South Florida, the family should definitely be called `SunshineJS`. It is the most inclusive to all the areas we want to cover. There could still be a meetup group called `MiamiJS` that's simply part of the `SunshineJS` family. Each individual meetup group and its leaders can choose the branding that most suits them. The `SunshineJS` family would be there for support, be it the need of meetup content, sponsorship, venue, whatever the case may be. Geographically speaking, I think that the `MiamiJS` brand makes the most sense for the group that @dambalah is spearheading. Node.js and browser JavaScript topics can be covered by the same group! If @dambalah wants to focus on Node.js, somebody else can take initiative for browser topics.

To @young-steveo's point, I do think `undefined` is a cool name. I'm thinking we could use it for meetup topics that aren't defined by JavaScript but are useful to all developers (editors, workflow, git, etc.).

@CristinaSolana After talking to @onel0p3z the other day he mentioned that you had some misgivings. Before we close the issue I'd love to hear what you have to say! We probably should have done this from the beginning since `MiamiJS` is your idea. I want all of us to feel good about what we are doing! Anyway, would you mind jumping on a hangout with @onel0p3z and I some time this week? @dambalah would also be welcome!

**Comment by CristinaSolana** [comment]
Hi @khalifenizar, I'd rather commit to Miami atm. Go ahead and create an org for the new name so we can transfer these repos over.

**Comment by nizaroni** [comment]
Hey guys! I've created the organization and sent invites! Here it is: https://github.com/SunshineJS.
